<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MapWindow</class>
 <widget class="QMainWindow" name="MapWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>12</y>
      <width>500</width>
      <height>500</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>10</y>
      <width>251</width>
      <height>111</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="moneyLable">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>20</y>
       <width>41</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Money</string>
     </property>
    </widget>
    <widget class="QLCDNumber" name="lcdFood">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>51</width>
       <height>23</height>
      </rect>
     </property>
    </widget>
    <widget class="QLCDNumber" name="lcdMoney">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>51</width>
       <height>23</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="foodLabel">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>50</y>
       <width>41</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Food</string>
     </property>
    </widget>
    <widget class="QLCDNumber" name="lcdWood">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>20</y>
       <width>51</width>
       <height>23</height>
      </rect>
     </property>
    </widget>
    <widget class="QLCDNumber" name="lcdStone">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>50</y>
       <width>51</width>
       <height>23</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="woodLabel">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>20</y>
       <width>41</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Wood</string>
     </property>
    </widget>
    <widget class="QLabel" name="stoneLabel">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>50</y>
       <width>41</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Stone</string>
     </property>
    </widget>
    <widget class="QLCDNumber" name="lcdOre">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>51</width>
       <height>23</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="oreLabel">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>80</y>
       <width>41</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Ore</string>
     </property>
    </widget>
    <widget class="QLabel" name="resourcesLabel">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>0</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Resources</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>270</y>
      <width>251</width>
      <height>191</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="hqButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>201</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>HQ</string>
     </property>
    </widget>
    <widget class="QPushButton" name="farmButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>150</y>
       <width>101</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Farm</string>
     </property>
    </widget>
    <widget class="QPushButton" name="sawButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>90</y>
       <width>101</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Saw Mill</string>
     </property>
    </widget>
    <widget class="QPushButton" name="outpustButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>101</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Outpost</string>
     </property>
    </widget>
    <widget class="QPushButton" name="mineButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>120</y>
       <width>101</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Mine</string>
     </property>
    </widget>
    <widget class="QPushButton" name="smallHouseButton">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>60</y>
       <width>101</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Small House</string>
     </property>
    </widget>
    <widget class="QPushButton" name="largeHouseButton">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>90</y>
       <width>101</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Large House</string>
     </property>
    </widget>
    <widget class="QPushButton" name="apartmentsButton">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>120</y>
       <width>101</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Apartments</string>
     </property>
    </widget>
    <widget class="QPushButton" name="skyscraperButton">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>150</y>
       <width>101</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Skyscraper</string>
     </property>
    </widget>
    <widget class="QPushButton" name="buildButton">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>0</y>
       <width>113</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Build</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_3">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>120</y>
      <width>251</width>
      <height>151</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="assignButton">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>110</y>
       <width>113</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Assign</string>
     </property>
    </widget>
    <widget class="QPushButton" name="unassignButton">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>110</y>
       <width>113</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Unassign</string>
     </property>
    </widget>
    <widget class="QPushButton" name="bwButton">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>20</y>
       <width>51</width>
       <height>41</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="default">
      <bool>false</bool>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="farmerButton">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>20</y>
       <width>51</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="default">
      <bool>false</bool>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="loggerButton">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>20</y>
       <width>51</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="default">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="minerButton">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>20</y>
       <width>51</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="default">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>41</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Total</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>41</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Free</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLCDNumber" name="lcdNumber">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>70</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
    </widget>
    <widget class="QLCDNumber" name="freeBwNumber">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>90</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
    </widget>
    <widget class="QLCDNumber" name="freeFarmerNumber">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>90</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
    </widget>
    <widget class="QLCDNumber" name="lcdNumber_4">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>70</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
    </widget>
    <widget class="QLCDNumber" name="freeLoggerNumber">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>90</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
    </widget>
    <widget class="QLCDNumber" name="lcdNumber_6">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>70</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
    </widget>
    <widget class="QLCDNumber" name="freeMinerNumber">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>90</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
    </widget>
    <widget class="QLCDNumber" name="lcdNumber_8">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>70</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>0</y>
       <width>131</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Worker Management</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="endTurnButton">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>480</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>End Turn</string>
    </property>
   </widget>
   <widget class="QPushButton" name="highScoreButton">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>510</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Highscore</string>
    </property>
   </widget>
   <widget class="QPushButton" name="quitButton">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>510</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Quit</string>
    </property>
   </widget>
   <widget class="QLabel" name="infoLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>520</y>
      <width>491</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="warningLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>540</y>
      <width>491</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
